tensornetwork/network_operations.py

```diff
@@ -899,3 +899,34 @@ def get_neighbors(node: BaseNode) -> List[Node]:
         neighbors.append(edge.node1)
         neighbors_set.add(edge.node1)
   return neighbors
+
+def kron(tensors: Sequence[Union[BaseNode, Tensor]]) -> BaseNode:
+  """Kronecker product of the given tensors/nodes.
+
+  Args:
+    tensors: A sequence of `Tensor`s or `BaseNode` objects.
+
+  Returns:
+    A `Node` that is the kronecker product of the given inputs. The first
+    half of the edges of this node would represent the "input" edges of the
+    operator and the last half of edges are the "output" edges of the
+    operator.
+  """
+  nodes = []
+  input_edges = []
+  output_edges = []
+  for tensor in tensors:
+    if isinstance(tensor, BaseNode):
```

Comment:
I think Node() also accepts `Node` as input, so the check here could probably be removed